xdrgen/src/lib.rs

```diff
@@ -107,19 +107,23 @@ pub fn generate<In, Out>(infile: &str, mut input: In, mut output: Out) -> Result
             })
             .map(|c| c.define(&xdr, e));
 
-        let typedefs = xdr.typedefs()
-            .map(|(n, ty)| spec::Typedef(n.clone(), ty.clone()))
+        let typespecs = xdr.typespecs()
+            .map(|(n, ty)| spec::Typespec(n.clone(), ty.clone()))
```

Comment:
What's the reasoning behind the typedef->typespec name? The term "typespec" doesn't appear in RFC4506, and I wanted to follow its terminology for the grammar.